fake_service/tests/test_api.py

```diff
@@ -6,47 +6,60 @@
 
 
 class SchemaApiTestCase(APITestCase):
-
     def setUp(self):
         self.user = User.objects.create(username="alabama", password="password")
         refresh = RefreshToken.for_user(self.user)
-        self.client.credentials(HTTP_AUTHORIZATION=f'Bearer {refresh.access_token}')
-        self.post = Schema.objects.create(user=self.user,
-                                          name="qwerty228",
-                                          column_separator=".",
-                                          string_character="'",
-                                          fields=[
-                                              {
-                                                  "name": "qwertty228",
-                                                  "type": "integer",
-                                                  "value_from": "20",
-                                                  "value_to": "34"
-                                              }
-                                          ])
+        self.client.credentials(HTTP_AUTHORIZATION=f"Bearer {refresh.access_token}")
+        self.post = Schema.objects.create(
+            user=self.user,
+            name="qwerty228",
+            column_separator=".",
+            string_character="'",
+            fields=[
+                {
+                    "name": "qwertty228",
+                    "type": "integer",
+                    "value_from": "20",
+                    "value_to": "34",
+                }
+            ],
+        )
 
     def test_post(self):
-        sample_post = {'name': 'Schema_2',
-                       'column_separator': ',',
-                       'string_character': '\'',
-                       'fields': [{"name": "Column_1",
-                                   "type": "integer",
-                                   "value_from": "1",
-                                   "value_to": "20"}]}
+        sample_post = {
+            "name": "Schema_2",
+            "column_separator": ",",
+            "string_character": "'",
+            "fields": [
+                {
+                    "name": "Column_1",
+                    "type": "integer",
+                    "value_from": "1",
+                    "value_to": "20",
+                }
+            ],
+        }
 
         response = self.client.post(reverse("schema-list"), sample_post, format="json")
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
status code check is fine. But you should check if new schema has been created in db as well.